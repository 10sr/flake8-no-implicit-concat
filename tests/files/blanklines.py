a = ["aaa"



     "bbb"]
b = ["aaa"  # comment



     "bbb"]
print(a)
