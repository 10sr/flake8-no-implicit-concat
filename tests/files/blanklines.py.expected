files/blanklines.py:1:11: NIC001 Implicitly concatenated string literals
files/blanklines.py:6:11: NIC001 Implicitly concatenated string literals
files/blanklines.py:13:15: NIC001 Implicitly concatenated string literals
